### PR TITLE
chore(polar_area_chart_plugin): upgrade to SDK 9

### PR DESCRIPTION
**Pull request by dremileta**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: polar_area_chart_plugin/src/dp_polar_area_chart_plugin/component/PolarAreaChart.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        .toArray()
+        .map((slice, i) => ({
+            name: slice.sliceTitles()[0],
+            value: parseFloat(`${slice.rawData()[0] ?? ""}`),
+            fill: colorPalette[i % colorPalette.length],
+        }));
+
+    // Couldn't figure out how to get month names so I hardcoded them 🤷‍♂️
+    // https://gooddata.slack.com/archives/C6PDGGNRH/p1682387905940879
+    // const labels = results.reduce((acc, r) => acc.concat(MONTH_NAMES[r.name]), []);
```
Comment (kazakovkirilliy) [inline polar_area_chart_plugin/src/dp_polar_area_chart_plugin/component/PolarAreaChart.tsx:62]: Could you please remove those comments (as well as the commented object above), it is not needed anymore in my opinion

**Comment by kazakovkirilliy** [comment]
Please, keep only one lock file.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by davidzoufaly** [comment]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-dashboard-plugins-gate-PR43

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-dashboard-plugins-sonarqube-scan-zuul-gate/43/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-dashboard-plugins-fossa-scanning-zuul/20/ : SUCCESS in 2m 04s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-dashboard-plugins-merge-compliance-check/44/ : SUCCESS in 9s

**Pull request merged**